package.json

```
@@ -3,32 +3,26 @@
   "version": "0.0.96",
   "description": "Standard library.",
   "license": "Apache-2.0 AND BSL-1.0",
-  "author": {
-    "name": "The Stdlib Authors",
-    "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
-  },
+  "author": "The Stdlib Authors (https://github.com/stdlib-js/stdlib/graphs/contributors)",
```

```suggestion
  "author": {
    "name": "The Stdlib Authors",
    "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
  },
```

This file should not be changed in this PR.